### PR TITLE
Add support for masking

### PR DESCRIPTION
**Pull request by yechenh**

### CODE REVIEW

File: lasertagger/bert/tokenization.py [hunk 1]
```diff
@@ -161,11 +161,15 @@ def whitespace_tokenize(text):
 class FullTokenizer(object):
   """Runs end-to-end tokenziation."""
 
-  def __init__(self, vocab_file, do_lower_case=True):
+  SPECIAL_SYMBOL_LIST = [".", ",", ";", "!", "?", ":",
```
Comment (XiangZeng2020) [inline lasertagger/bert/tokenization.py:164]: Either define this in the global scope, or define it as a variable of the class: self.special_symbol_list = ...
https://google.github.io/styleguide/pyguide.html#25-global-variables

If this is internal to the module, add a '_' prefix.

File: lasertagger/bert/tokenization.py [hunk 2]
```diff
@@ -161,11 +161,15 @@ def whitespace_tokenize(text):
 class FullTokenizer(object):
   """Runs end-to-end tokenziation."""
 
-  def __init__(self, vocab_file, do_lower_case=True):
+  SPECIAL_SYMBOL_LIST = [".", ",", ";", "!", "?", ":", 
+                               "##.", "##,", "##;", "##!", "##?", "##:"]
+  
+  def __init__(self, vocab_file, do_lower_case=True, do_masking=False):
```
Comment (XiangZeng2020) [inline lasertagger/bert/tokenization.py:167]: s/do_lower_case/lower_case
s/do_masking/enable_mask

avoid "do*" in general which is not common in google codebase.

File: lasertagger/bert/tokenization.py [hunk 3]
```diff
@@ -161,11 +161,15 @@ def whitespace_tokenize(text):
...
     self.vocab = load_vocab(vocab_file)
     self.inv_vocab = {v: k for k, v in self.vocab.items()}
```
Comment (XiangZeng2020) [inline lasertagger/bert/tokenization.py:169]: what does inv_vocab mean? use more meaningful name

File: lasertagger/bert_example.py [hunk 4]
```diff
@@ -123,10 +123,12 @@ def __init__(self, label_map, vocab_file,
         uncased models and False for cased models.
       converter: Converter from text targets to tags.
       embedding_type: POS or Normal or Sentence.
+      do_masking: whether to mask numbers and symbols
     """
     self._label_map = label_map
     self._tokenizer = tokenization.FullTokenizer(vocab_file,
-                                                 do_lower_case=do_lower_case)
+                                                 do_lower_case=do_lower_case,
```
Comment (XiangZeng2020) [inline lasertagger/bert_example.py:130]: s/do_lower_case/to_lower_case
s/do_masking/enable_mask
ditto

File: lasertagger/bert/tokenization.py (hunk 3 above)
Comment (yechenh) [reply lasertagger/bert/tokenization.py:169]: This is part of the original bert code, and I do not intend to use this part.

File: lasertagger/bert/tokenization.py (hunk 1 above)
Comment (yechenh) [reply lasertagger/bert/tokenization.py]: Done.

File: lasertagger/bert/tokenization.py (hunk 2 above)
Comment (yechenh) [reply lasertagger/bert/tokenization.py]: Done.

File: lasertagger/bert_example.py (hunk 4 above)
Comment (yechenh) [reply lasertagger/bert_example.py]: Done.

**Pull request merged**